scripts/push-to-s3-prod.sh

```diff
@@ -7,7 +7,7 @@ export ALGOLIA_INDEX_NAME="docs_site_prod"
 export ALGOLIA_API_KEY="49a495ba4c210780a28feed306d05522" # This is a search only key. It is safe to check in.
 
 export CONFIG_POSTHOG_API_KEY="phc_zJYg43ctbPl1BzsnODjop1op9KSsctiGm6DNNt3zHZT"
-export CONFIG_POSTHOG_APP_URL="https://docs.gruntwork.io"
```

Comment:
I considered this but worried it would break stage because the posthogConfig would exist but there will no apiKey.